tests/moduleTests/stream.ts

```diff
@@ -521,6 +521,20 @@ export default function tests() {
         assert.ok(n2.pitch.accidental.displayStatus);
     });
 
+    test('music21.stream.Stream makeAccidentals superfluous naturals', assert => {
+        const n1 = new music21.note.Note('En4');
+        const m = new music21.stream.Measure();
+        m.append(n1);
+        m.makeAccidentals({inPlace: true});
+        assert.notOk(n1.pitch.accidental.displayStatus);
+
+        n1.pitch.accidental.displayStatus = undefined;
+        const n0 = new music21.note.Note('E4');
+        m.insertAndShift(0, n0);
+        m.makeAccidentals({inPlace: true});
+        assert.notOk(n1.pitch.accidental.displayStatus);
```

Comment:
I think we need tests with keySignatures before going forward.

Eb-major;  Ab clicked: accidentalDisplayStatus is false.   # clicked for A clicked: accidental display status is true;  natural clicked: still true; flat clicked: accidental displayStatus is none

C-natural (in Eb-major).  accidental is undefined;  sharp clicked: a.d.s. = true. natural clicked again -- accidental probably still there as natural, but displayStatus is also none.